ecosystem/sep-0031.md

```diff
@@ -346,14 +346,14 @@ Name | Type | Description
 `amount_in` | string | (optional) The amount received or to be received by the Receiving Anchor. Up to 7 decimals are permitted. Excludes any fees charged after Receiving Anchor receives the funds.
 `amount_out` | string | (optional) The amount sent by the Rending Anchor to Receiving Client. Up to 7 decimals are permitted.
 `amount_fee` | string | (optional) The amount of fee charged by the Receiving Anchor.
-`stellar_account_id` | string | The Stellar account to send the payment to.
+`stellar_account_id` | string | The Receiving Anchor's Stellar account the Sending Anchor will be making the payment to.
```

Comment:
💡 I could be wrong, but when I read this my head automatically inserted this word between these two, then I realized it wasn't there. 
```suggestion
`stellar_account_id` | string | The Receiving Anchor's Stellar account that the Sending Anchor will be making the payment to.
```